Automattic/jetpack
# Fix failing Jetpack plugin JS Tests

Following on from https://github.com/Automattic/jetpack/pull/20512 which introduced Jetpack Boost, it appears that the Jetpack plugin JS tests have been failing (for extensions / blocks in particular). This seems to be to do with a version mismatch between the Jest version that the Jetpack plugin / Jetpack extensions tests were expecting and the one that was running. This PR updates the Jest dependency  for the Jetpack plugin to > 27.

The helpful links I found while investigating the failing tests were:

* https://github.com/facebook/jest/issues/11444#issuecomment-847909076
* https://github.com/facebook/jest/issues/7868

#### Screenshots

Before:

![image](https://user-images.githubusercontent.com/14988353/127809380-2cb02a2d-aaed-4cb8-bdfd-8e4ad96696ec.png)

After:

![image](https://user-images.githubusercontent.com/14988353/127809400-23233991-980f-42dd-aa35-144d651df531.png)

#### Changes proposed in this Pull Request:

* Update Jest dependency for the Jetpack plugin to fix failing JS tests

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* See if the CI JS test jobs complete, or to test locally:
* Go to `projects/plugins/jetpack` and run `pnpm test-extensions` to run tests. Check that they pass!
* Check that this hasn't caused any other tests / jobs to fail
